### PR TITLE
upgrade testcontainers to 1.16.3

### PR DESCRIPTION
**Pull request by KyleAure**
#build

### CODE REVIEW

**Review by gjwatts — APPROVED** [review]

**Comment by KyleAure** [comment]
#build

**Comment by KyleAure** [comment]
#build

**Comment by KyleAure** [comment]
Looks like this update will break the Selenium UI tests.  Going to work with @amyreit to work on a matching PR to ensure the UI tests are updated for the new selenium version.

**Comment by KyleAure** [comment]
#build

**Comment by KyleAure** [comment]
#build

File: dev/cnf/oss_dependencies.maven [hunk 1]
```diff
@@ -602,7 +602,9 @@ org.rnorth.duct-tape:duct-tape:1.0.8
 org.rnorth.visible-assertions:visible-assertions:2.1.2
 org.seleniumhq.webdriver:webdriver-common:0.9.7376
 org.slf4j:slf4j-api:1.7.30
+org.slf4j:slf4j-api:1.7.32
 org.slf4j:slf4j-simple:1.7.30
+org.slf4j:slf4j-simple:1.7.32
```
Comment (tevans78) [inline dev/cnf/oss_dependencies.maven]: please don't duplicate dependency versions ... update the existing one and move up to the latest version available

Comment (tevans78) [reply dev/cnf/oss_dependencies.maven]: latest version is 1.7.36

File: dev/com.ibm.ws.app.manager.wab.installer_fat/build.gradle [hunk 2]
```diff
@@ -10,7 +10,7 @@
  *******************************************************************************/
  
 dependencies {
-  requiredLibs 'org.apache.commons:commons-compress:1.20'
+  requiredLibs 'org.apache.commons:commons-compress:1.21'
```
Comment (tevans78) [inline dev/com.ibm.ws.app.manager.wab.installer_fat/build.gradle:13]: given how many places that commons-compress is used, we should probably create a test bundle for it and depend on that... `io.openliberty.org.apache.commons.compress`

**Comment by tevans78** [comment]
I ran three personal builds. One of them did not have any UI failures. The other two had existing known UI failures. This is the "good" build: 
https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemOid/com.ibm.team.build.BuildResult/_NF0jgLMfEey_l6cwP43xMA

**Comment by tevans78** [comment]
#libby

**Pull request merged**